components/item-info.js

```diff
@@ -122,6 +122,8 @@ export default function ItemInfo ({
         <Link href={`/~${item.subName}`}>
           {' '}<Badge className={styles.newComment} bg={null}>{item.subName}</Badge>
         </Link>}
+      {sub.nsfw &&
```

Comment:
https://github.com/stackernews/stacker.news/pull/788/commits/84d95451311e7a6b123765b95abd6c9ac3563849

that's good to know. i was wondering whether null subs were expected since we were doing left joins for them instead of inner joins so that makes sense.